datascience/utils.py

```diff
@@ -41,3 +43,31 @@ def get_production(countries, start_date, end_date, delta):
                               'production': o['production'].values()})
             df = df.append(p)
     return df
+
+# get_exchange
+
+def fetch_exchange(country_code, t):
+    endpoint = 'http://electricitymap.tmrow.co'
+    url = '%s/v1/state' % endpoint
```

Comment:
We should make a specific `/v1/exchange` route. The state route runs the CO2 model, and we probably don't want that? I can help with that if you wish. Else, it should be quite self-explanatory in `server.js`.